src/utils.js

```diff
@@ -60,3 +62,26 @@ export function recalculateActive (terminalsSet, active) {
 		t.emitter.emit('did-change-title')
 	})
 }
+
+// finds the default shell start commmand
+export function getShellStartCommand () {
```

Comment:
LGTM. Could you add tests to `spec/utils-spec.js` for each path in this function?

Reply:
The output of the function depends on the software that is installed on the system, and it is not predictable. I do not think we can test it.

Reply:
You should be able to mock the system output

```js
import childProcess from 'child_process'

spyOn(childProcess, "spawn").and.callFake(() => {})
...
spyOnProperty(process.env, "COMSPEC").and.returnValue('comspec') // 
```

Reply:
I added a commit with the tests